src/test/javascript/spec/e2e.ts

```diff
@@ -77,4 +92,30 @@ describe('Screenshot Tests', () => {
     expect(await mutationList.isDisplayed()).toBe(false);
     expect(singleMutationView).toExist();
   });
+
+  it('should delete mutation immediately if newly created and not reviewed', async () => {
+    await browser.url(`${BASE_URL}/curation/EMPTYGENE/somatic`);
+
+    // Click to open mutation modal
+    const addMutationButton = await $('button=Add Mutation');
+    await addMutationButton.waitForDisplayed();
+    await addMutationButton.click();
+
+    await createMutationOnCurationPage('V600E');
+
+    // Delete the mutation on curation page
+    const mutationDeleteBtn = await $("div[data-testid='V600E-collapsible']").$("svg[data-icon='trash-can']");
```

Comment:
we need a testid generator to generate the id here. The knowledge of this id pattern is easy for us, but may not be to a new engineer without the context.